libs/cimit-service/src/main/java/uk/gov/di/ipv/core/library/service/CiMitService.java

```diff
@@ -77,7 +77,8 @@ public class CiMitService {
     private static final String POST_CI_ENDPOINT = "/contra-indicators/detect";
     private static final String POST_MITIGATIONS_ENDPOINT = "/contra-indicators/mitigate";
     private static final String GET_VCS_ENDPOINT = "/contra-indicators";
-    private static final String DUMMY_CIMIT_API_KEY = "dummyApiKey";
+    private static final String PLACEHOLDER_CIMIT_API_KEY =
```

Comment:
Tempted to call this constant `NOT_REQUIRED` - keeps the code where it's used a bit more understandable